.github/workflows/format.yml

```
@@ -0,0 +1,41 @@
+name: Format Check
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)